test/src/test/java/hudson/model/JobPropertyTest.java

```diff
@@ -115,7 +133,13 @@ public JobPropertyWithConfigImpl(String name) {
         }
 
         @TestExtension("configRoundtrip")
-        public static class DescriptorImpl extends JobPropertyDescriptor {}
+        public static class DescriptorImpl extends JobPropertyDescriptor {
+            @Override
+            public JobProperty<?> newInstance(StaplerRequest req, JSONObject formData) throws FormException {
```

Comment:
Shouldn't this be a separate test? Looks like it loses coverage of the case that `newInstance` is not overridden.